test/src/main/java/org/apache/accumulo/test/mapred/AccumuloRowInputFormatIT.java

```diff
@@ -31,13 +31,15 @@
 import org.apache.accumulo.core.client.BatchWriter;
 import org.apache.accumulo.core.client.BatchWriterConfig;
 import org.apache.accumulo.core.client.MutationsRejectedException;
-import org.apache.accumulo.core.client.mapred.AccumuloRowInputFormat;
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.KeyValue;
 import org.apache.accumulo.core.data.Mutation;
 import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.security.Authorizations;
 import org.apache.accumulo.core.security.ColumnVisibility;
 import org.apache.accumulo.core.util.PeekingIterator;
+import org.apache.accumulo.hadoop.mapred.AccumuloRowInputFormat;
+import org.apache.accumulo.hadoop.mapreduce.InputInfo;
```

Comment:
This test need not be modified. It's currently testing the core mapreduce stuff, and should continue to do so. Rather than mod, these mapred tests should be copied and modified in the new module's `src/test/java` directory to run there.